lldb/test/API/lang/swift/async/formatters/task/children/main.swift

```diff
@@ -0,0 +1,10 @@
+func f() async -> Int {
+    return 30
+}
+
+@main struct Main {
+    static func main() async {
+        async let number = f()
+        await print("break here \(number)")
```

Comment:
Is this a race condition if f() already finished by the time we hit the breakpoint, and does it need another very long sleep()?

Reply:
good catch, I'll add a sleep.